### ISSUE TITLE
Parallel query example is confusing

### ISSUE THREAD

**Issue by pllim** [astropy/astroquery#846]
http://astroquery.readthedocs.io/en/latest/api.html#parallel-queries
```python
from astroquery import module

QC = QueryClass(login_information)

results = parallel_map(QC.query_object,['m31','m51','m17'],radius=['1"','1"','1"'])

results = [QC.query_object_async(obj, radius=r)
    for obj,r in zip(['m31','m51','m17'],['1"','1"','1"'])]
```

While I understand this may be pseudo-code, I still cannot follow it. For example, why is `module` imported but unused? Where do `QueryClass` and `parallel_map` come from?

Is your goal to demonstrate that `parallel_map` and `query_object_async` are interchangeable? Why are there so many ways to accomplish this?

**Comment by bsipocz** [comment]
I have never seen this in use, but would guess that ``QueryClass`` is actually comes from the module, e.g.:
``from astroquery.eso import Eso`` where ``Eso`` is in fact an ``EsoClass`` object.

**Comment by keflavich** [comment]
Yeah, I think the example should have:
`from astroquery.module import QueryClass`

And then `parallel_map` should be imported from somewhere else; it is meant to be "pure" pseudo code, i.e., it is some parallel implementation of the `map` function.

**Comment by AyushYadav** [comment]
@bsipocz  Fixed in PR #856
